README.md

```diff
@@ -91,6 +91,30 @@ ZPC> zwave_add_node
 (...)
 ```
 
+### WSL2: Mount devices from windows
```

Comment:
I would just add 

WSL2 Build

as chapter title to align other

there you can share hints